### PR TITLE
feat(raml): add "initialAccountValue" to the account information

### PR DESCRIPTION
**Pull request by abuecker**

### CODE REVIEW

**Comment by abuecker** [comment]
@johntowell please review

File: api/v3/examples/profile.res.json [hunk 1]
```diff
@@ -94,6 +95,7 @@
       "status": "CLOSED",
       "dateOpened": "2015-10-20T00:42:41.617Z",
       "dateClosed": "2013-10-07T00:00:00.000-04:00",
+      "initialAccountValue": "94.54",
       "accountValue": "67056.68",
```
Comment (eweise) [inline api/v3/examples/profile.res.json]: why string type for amount?

Comment (johntowell) [inline api/v3/examples/profile.res.json]: not sure why i had the numbers as strings, should we change?

**Comment by johntowell** [comment]
👍

File: api/v3/examples/profile.res.json (hunk 1 above)
Comment (abuecker) [inline api/v3/examples/profile.res.json]: I'll change them to numbers.

**Comment by abuecker** [comment]
@bump Changed amount types from string to number.

**Comment by johntowell** [comment]
👍

**Comment by johntowell** [comment]
wait though, this will break platform, need to synchronize the changes

**Comment by abuecker** [comment]
Since there isn't a good way to address this right now, let's let it break.  I'll post in the developer room that it will happen.

**Comment by johntowell** [comment]
PR for platform up

https://github.com/trizic/platform/pull/586

**Pull request merged**